### PR TITLE
Process specified device sps nal data

### PR DESCRIPTION
**Pull request by spiritedRunning**
Test in p7 phone.

### CODE REVIEW

File: lib/src/main/java/net/ypresto/androidtranscoder/utils/AvcCsdUtils.java [hunk 1]
```diff
@@ -29,6 +29,8 @@
     private static final byte[] AVC_START_CODE_4 = {0x00, 0x00, 0x00, 0x01};
     // Refer: http://www.cardinalpeak.com/blog/the-h-264-sequence-parameter-set/
     private static final byte AVC_SPS_NAL = 103; // 0<<7 + 3<<5 + 7<<0
+    // https://tools.ietf.org/html/rfc6184
+    private static final byte AVC2_SPS_NAL = 39; // 0<<7 + 1<<5 + 7<<0
```
Comment (ypresto) [inline lib/src/main/java/net/ypresto/androidtranscoder/utils/AvcCsdUtils.java]: Also `2<<5` could be happen

Comment (ypresto) [inline lib/src/main/java/net/ypresto/androidtranscoder/utils/AvcCsdUtils.java]: Please rename AVC2_SPS_NAL -> AVC_SPS_NAL_2

**Comment by ypresto** [comment]
Fixes #27

**Comment by spiritedRunning** [comment]
I have re-commit, please review.

**Comment by ypresto** [comment]
Also working perfectly for Nexus 5X.
Thanks!!

LGTM
:+1:

**Pull request merged**